spec/services/paypal_service/store/paypal_payment_spec.rb

```diff
@@ -1,6 +1,5 @@
 describe PaypalService::Store::PaypalPayment do
   order = {
-    order_total: 100,
```

Comment:
This fixes a warning also. There's a `order_total` redefined in line 12, and that line overrides this definition.